test/functional/http/post/0.tx.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+var node = require('../../../node');
+var shared = require('./shared');
+var constants = require('../../../../helpers/constants');
+
+var sendTransaction = require('../../../common/complexTransactions').sendTransaction;
+
+describe('POST /api/transactions (type 0)', function () {
+
+	var badTransactions = [];
+	var goodTransactions = [];
+
+	describe('schema checks', function () {
+
+		var transaction;
+
+		beforeEach(function () {
+			transaction = node.randomTx();
+		});
+
+		it('using null should fail', function (done) {
+			sendTransaction(null, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using undefined should fail', function (done) {
+			sendTransaction(undefined, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using NaN should fail', function (done) {
+			sendTransaction(NaN, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using integer should fail', function (done) {
+			sendTransaction(0, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using empty string should fail', function (done) {
+			sendTransaction('', function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using empty array should fail', function (done) {
+			sendTransaction([], function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+
+		it('using empty object should fail', function (done) {
+			sendTransaction({}, function (err, res) {
+				node.expect(res).to.have.property('success').not.to.be.ok;
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Empty trs passed');
+				done();
+			}, true);
+		});
+	});
+
+	describe('processing', function () {
```

Comment:
This is a new convention. Change the description of the issue if you prefer another way.